nidirect_breadcrumbs/src/SearchBreadcrumb.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Drupal\nidirect_breadcrumbs;
+
+/**
+ * @file
+ * Generates the breadcrumb trail for search page(s)
+ */
+
+use Drupal\Core\Breadcrumb\Breadcrumb;
+use Drupal\Core\Breadcrumb\BreadcrumbBuilderInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+class SearchBreadcrumb implements BreadcrumbBuilderInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function applies(RouteMatchInterface $route_match) {
+    $match = FALSE;
+
+    if ($route_match->getRouteName() == 'view.search.page_1') {
```

Comment:
I'd be tempted to pass the required matches in as params rather than restrict it hardcoded values in the code.

Reply:
Just want to check:

> pass the required matches in as params

Were you thinking of config, or something else? Quite likely I'm brain-farting and missing something in plain-sight :)

Reply:
Yeah, pass them in as params in your services.yml file.

Reply:
Ah, cool. I haven't ever needed to use those before, that's pretty handy. Nice one.

I've pushed up a fresh commit with the change